docs/external-db.md

```diff
@@ -0,0 +1,118 @@
+# External DB integration
+
+Backstage hosts the data in a [PostgreSQL database](https://backstage.io/docs/getting-started/config/database/).
+By default, the Operator creates and manages a local instance of PostgreSQL in the same namespace as the Backstage deployment but it also allows to switch this off and configure an external database server instead.
+Usually, external connection requires more security, so, this instruction includes steps to configure SSL/TLS.
+
+### Configure your external PostgreSQL instance
+As a prerequisite, you have to know:
+- **db-host** - your PostgreSQL instance DNS or IP address 
+- **db-port** - your PostgreSQL instance port number (usually 5432)
+- **username** - to connect to your PostgreSQL instance
+- **password** - to connect to your PostgreSQL instance
+
+**NOTE:** By default, Backstage uses databases for each plugin and automatically creates them if none are found, so in addition to PSQL Database level privileges, the user may need Create Database privilege.  
+
+In addition, to get your database connection secured with SSL/TLS, you also need certificates in the form of PEM file. 
+
+You can find configuration guidelines for:
+- [AWS RDS PostgreSQL](https://github.com/janus-idp/operator/blob/main/docs/external-db.md#aws-rds-postgresql)
+- [Azure Database PostgreSQL](https://github.com/janus-idp/operator/blob/main/docs/external-db.md#aws-rds-postgresql)
+
+If you want to move Backstage database from local to external, here is a [Migration Guide](https://github.com/janus-idp/operator/blob/main/docs/db_migration.md).
+
+### Create secret with PostgreSQL connection properties:
+````yaml
+cat <<EOF | kubectl -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: <cred-secret>
+type: Opaque
+stringData:
+ POSTGRES_PASSWORD: <password>
+ POSTGRES_PORT: <db-port>
```

Comment:
Just to clarify that this needs to be a string. Otherwise, it won't work with just the port number.

```suggestion
 POSTGRES_PORT: "<db-port>"
```